libafl_libfuzzer/Cargo.toml

```diff
@@ -35,6 +35,9 @@ introspection = []
 ## `-fsanitize=fuzzer-no-link -l:libafl_libfuzzer_runtime.a`
 embed-runtime = []
 
+## 🐇
```

Comment:
I'd still give some extra context here to not confuse ppl checking out available features? Or is it really just meant as easteregg? :D

Reply:
Oh, the rabbit mode just changes the namespace name. It will always be put into a separate namespace (with rabbit mode disabled the namespace is `__libafl`.

Reply:
Literal easteregg

Reply:
:rabbit2: